recipes/ckanweb-deploy.rb

```diff
@@ -301,13 +301,21 @@
 	mode "0755"
 end
 
-file "/etc/cron.daily/ckan-tracking-update" do
-	content "/usr/local/bin/pick-job-server.sh && #{paster} tracking update -c #{config_file} 2>&1 >/dev/null\n"
-	owner "root"
+# Remove unwanted cron job from previous script versions
+#
+file '/etc/cron.daily/ckan-tracking-update' do
+	action :delete
+end
+
+# Run tracking update at 8:30am and 6:30pm
+file "/etc/cron.d/ckan-tracking-update" do
+	content "30 8,18 * * * root /usr/local/bin/pick-job-server.sh && #{paster} tracking update -c #{config_file} 2>&1 >/dev/null\n"
```

Comment:
Perhaps it would be simpler to just run it hourly?

Reply:
will be on lower environments also to help in testing